plugins/generator-1.20.1/forge-1.20.1/mappings/projectiles.yaml

```diff
@@ -1,40 +1,52 @@
 _default:
   - Snowball
   - EntityType.SNOWBALL
+  - Items.AIR
 _mcreator_prefix: "CUSTOM:"
 _mcreator_map_template:
   - "@NAMEEntity"
   - "@JavaModNameEntities.@REGISTRYNAME.get()"
+  - "@NAMEEntity.PROJECTILE_ITEM.getItem()"
 Arrow:
   - Arrow
   - EntityType.ARROW
+  - Items.ARROW
 DragonFireball:
   - DragonFireball
   - EntityType.DRAGON_FIREBALL
+  - Items.AIR
 Egg:
   - ThrownEgg
   - EntityType.EGG
+  - Items.AIR

```

Comment:
Shouldn't there be an egg item?

Reply:
https://github.com/MCreator/MCreator/pull/3456#discussion_r1252240648

Reply:
I think the list should be correct even now though, I think it is better to make it correct now than keep it with air which is a bit confusing until things from the linked comment are implemented.

Or is there a downside?

Reply:
> Or is there a downside?

The system (that is (should be if I made everything correctly) compatible for for the future) will simply break and might cause more problems. I wanted to make a system already capable, or as most as possible, of supporting more types of projectile in the future without having to change everything again. Using air was the easiest way I found to avoid making even more changes.

Reply:
What will break in this PR if you already set egg to Items.EGG now?